src/index.html

```diff
@@ -39,25 +36,25 @@ <h2 class="subtitle">Recent scores (Quidditch Game)</h2>
             </div>
             <div>
                 <h3 class="subtitle">Add your score</h3>
-                <form action="POST" class="add-score-form">
+                <form action="POST" class="add-score-form" autocomplete="off">
                     <label class="add-score-label" for="name">
-                        <input class="add-score-input" type="text" id="name" name="Your name" required
+                        <input class="add-score-input" type="text" id="name" name="Your name" required placeholder="Your name"
```

Comment:
- [ ] OPTIONAL
The _Your name_ input field is able to accept numbers whiles _Your score_ input field accepts letters. Kindly endeavor to provide input validations for the same.